packages/react/src/components/Tabs/Tab.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { useDidUpdate } from '../../index';
+
+interface TabProps {
```

Comment:
```suggestion
interface TabProps extends React.HTMLAttributes<HTMLLIElement> {
```

Allow specific attributes to be added to the element